<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>501</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="objectPathTextField"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="texturePathTextField"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="objSelectButton">
       <property name="text">
        <string>Select Object</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pngSelectButton">
       <property name="text">
        <string>Select Texture</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="heightLineEdit">
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="widthLineEdit">
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Angle:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="angleLineEdit">
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="renderButton">
       <property name="text">
        <string>Render</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="rotation0">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotation45">
    <property name="geometry">
     <rect>
      <x>373</x>
      <y>142</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotation90">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotation135">
    <property name="geometry">
     <rect>
      <x>373</x>
      <y>258</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotation180">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotation225">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>258</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotation270">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotation315">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>142</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rotationsMainLabel">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>203</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotations</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotationLabel0">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>97</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotationLabel45">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>135</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>45</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotationLabel90">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>202</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>90</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotationLabel135">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>25</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>135</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotationLabel180">
    <property name="geometry">
     <rect>
      <x>308</x>
      <y>307</y>
      <width>25</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>180</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotationLabel225">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>270</y>
      <width>25</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>225</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotationLabel270">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>202</y>
      <width>25</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>270</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotationLabel315">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>135</y>
      <width>25</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>315</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
